arch/arm/core/aarch32/nmi.c

```diff
@@ -91,5 +91,5 @@ void z_NmiHandlerSet(void (*pHandler)(void))
 void z_arm_nmi(void)
 {
 	handler();
-	z_arm_exc_exit();
+	z_arm_int_exit();
```

Comment:
@stephanosio this is a no-op right?

Reply:
For Cortex-M, `z_arm_exc_exit` and `z_arm_int_exit` are the same so it does not make any difference.

For Cortex-A/-R, with the changes in this PR, it must be `z_arm_int_exit` because the "NMI" is currently wired to FIQ and they must be handled in the same manner as IRQs (I am eventually going to rework this as well since it is not really correct).

Reply:
got it thanks